### PR TITLE
Enable NetApp driver for Cinder

### PR DESCRIPTION
**Pull request by rhefner1**
Enables a user to use NetApp storage systems
as backends for Cinder.

### CODE REVIEW

File: app/models/staypuft/deployment/cinder_service.rb [hunk 1]
```diff
@@ -134,6 +202,14 @@ def lvm_ptable
       end
     end
 
+    %w{hostname login password server_port storage_familie transport_type storage_protocol
```
Comment (rhefner1) [inline app/models/staypuft/deployment/cinder_service.rb]: @bcrochet does this change look right?

Comment (bcrochet) [inline app/models/staypuft/deployment/cinder_service.rb]: That looks correct.. Will test it out.

Comment (rhefner1) [inline app/models/staypuft/deployment/cinder_service.rb]: Any update on this? The goal is to get this into Staypuft ASAP and then on to OSP 6.

Comment (japplewhite) [inline app/models/staypuft/deployment/cinder_service.rb]: Status on this? We would really like to get this code merged asap.

**Comment by bcrochet** [comment]
LGTM.

**Pull request merged**